Public/Src/Tools/Xldb/XldbDataStore.cs

```diff
@@ -55,7 +53,7 @@ public sealed class XldbDataStore : IDisposable
             else
             {
                 Accessor = null;
-                Console.Error.WriteLine("Could not create an accessor for RocksDB. Accessor is null");
+                Console.Error.WriteLine("Could not create an accessor for RocksDB. Accessor is null!!! " + accessor.Failure.DescribeIncludingInnerFailures());

```

Comment:
Now that we fixed, we don't need to shout so much :)

Perhpas a single . will be enough ;)